destroy_cluster.sh

```diff
@@ -12,38 +13,63 @@ if [ ! -r "$CONFIG" ]; then
 fi
 source ./${CONFIG}
 
-if [[ $1 == '--force' ]]; then
+for i in "$@"
+do
+case $i in
+    -f|--force)
+    FORCE=true
+    shift
+    ;;
+    -i=*|--infra-id=*)
```

Comment:
If we're going to add more command line options to this script we should probably consider using something more standard like getopts.

For instance, while testing this, I passed `destroy_cluster.sh -i cluster_name` and it didn't work, it expected it in the format `destroy_cluster.sh -i=cluster_name`. Command line programs usually accept both syntaxes.

Reply:
done